src/actions.rs

```diff
@@ -107,28 +108,53 @@ pub enum PointerAction {
     },
     /// Pointer button down.
     Down {
-        /// The mouse button index. Button values are as follows:
-        /// - Left = 0
-        /// - Middle = 1
-        /// - Right = 2
+        /// The mouse button index.
+        ///
+        /// The following constants are provided:
```

Comment:
Let's link to them with `[]`, and also add a note saying "but any mouse index can be used to represent the corresponding mouse button".